app/controllers/users_controller.rb

```diff
@@ -14,7 +14,9 @@ def create
     using_recaptcha = !params[:spamaway] && Rails.env == "production"
     recaptcha = verify_recaptcha(model: @user) if using_recaptcha
     @spamaway = Spamaway.new(spamaway_params) unless using_recaptcha
-    if ((@spamaway&.valid?) || recaptcha) && @user.save
+
+    saved = @user.save
+    if ((@spamaway&.valid?) || recaptcha) && saved
```

Comment:
If redundant remove it

Reply:
The refactoring is important because it allows `@user.save` to be called everytime, populating the errors object.

Reply:
I explained this in more detail here, https://github.com/publiclab/plots2/pull/6932#issue-350216461

Reply:
Can we shift this to another PR please